invenio_oaiharvester/manage.py

```diff
@@ -28,11 +28,13 @@
     -f --from <lower bound date for selective harvesting>
     -t --to <upper bound date for selective harvesting>
     -u --url <the url for the OAI repo to harvest>
-    -o --output_dir <>
-    -w --workflow <>
+    -o --output <type of the output: workflow, dir/directory or stdout (default)>
+    -w --workflow <the name of the workflow to be used>
+    -d --dir <the directory where the harvested records should be saved>
 """
 from invenio.ext.script import Manager
 from .errors import IdentifiersOrDates
+from invenio_oaiharvester.config import OAIHARVESTER_STORAGEDIR
```

Comment:
You'd wanna use `invenio.base.globals:cfg` to get the value of `OAIHARVESTER_STORAGEDIR`. Otherwise it would be impossible to override (it would always take the default value from `config.py`.